tests/providers/microsoft/azure/hooks/test_asb_admin_client.py

```diff
@@ -0,0 +1,104 @@
+            conn_type='azure_service_bus',
+            login=self.client_id,
+            password=self.secret_key,
+            extra=json.dumps({"connection_string": self.connection_string}),
+        )
+
+    @mock.patch(
+        "airflow.providers.microsoft.azure.hooks.asb_admin_client."
+        "AzureServiceBusAdminClientHook.get_connection"
+    )
+    def test_get_conn(self, mock_connection):
+        mock_connection.return_value = self.mock_conn
+        hook = AzureServiceBusAdminClientHook(azure_service_bus_conn_id=self.conn_id)
+        assert isinstance(hook.get_conn(), ServiceBusAdministrationClient)
+
+    @mock.patch('azure.servicebus.management.QueueProperties')
+    @mock.patch(
+        'airflow.providers.microsoft.azure.hooks.asb_admin_client.AzureServiceBusAdminClientHook.get_conn'
+    )
+    def test_create_queue(self, mock_sb_admin_client, mock_queue_properties):
+        """
+        Test `create_queue` hook function with mocking connection, queue properties value and
+        the azure service bus `create_queue` function
+        """
+        mock_queue_properties.name = self.queue_name
+        mock_sb_admin_client.return_value.__enter__.return_value.create_queue.return_value = (
+            mock_queue_properties
+        )
+        hook = AzureServiceBusAdminClientHook(azure_service_bus_conn_id=self.conn_id)
+        response = hook.create_queue(self.queue_name)
+        assert response == mock_queue_properties
+
+    @mock.patch('airflow.providers.microsoft.azure.hooks.asb_admin_client.ServiceBusAdministrationClient')
+    def test_create_queue_exception(self, mock_sb_admin_client):
+        """
+        Test `create_queue` functionality to raise AirflowException
+        by passing queue name as None and pytest raise Airflow Exception
+        """
+        hook = AzureServiceBusAdminClientHook(azure_service_bus_conn_id=self.conn_id)
+        with pytest.raises(AirflowException):
+            hook.create_queue(None)
+
+    @mock.patch(
+        'airflow.providers.microsoft.azure.hooks.asb_admin_client.AzureServiceBusAdminClientHook.get_conn'
+    )
+    def test_delete_queue(self, mock_sb_admin_client):
+        """
+        Test Delete queue functionality by passing queue name, assert the function with values,
+        mock the azure service bus function  `delete_queue`
+        """
+        hook = AzureServiceBusAdminClientHook(azure_service_bus_conn_id=self.conn_id)
+        hook.delete_queue(self.queue_name)
+        expected_calls = [mock.call().__enter__().delete_queue(self.queue_name)]
+        mock_sb_admin_client.assert_has_calls(expected_calls)
+
+    @mock.patch('airflow.providers.microsoft.azure.hooks.asb_admin_client.ServiceBusAdministrationClient')
+    def test_delete_queue_exception(self, mock_sb_admin_client):
+        """
+        Test `delete_queue` functionality to raise AirflowException,
+         by passing queue name as None and pytest raise Airflow Exception
```

Comment:
```suggestion
         by passing queue name as None and checking whether AirflowException is raised
```

Reply:
corrected ` Test Delete queue functionality by passing queue name, assert the function with values,
        mock the azure service bus function  `delete_queue``